scripts/generate/_collect_symbols_build.sh

```diff
@@ -20,11 +20,7 @@ function cmake_build_options() {
 }
 
 function filter_symbols() {
-  if [[ "${GENERATE_FIPS}" -eq 0 ]]; then
-    grep -v "^_\?bignum_" | grep -v "_\?curve25519_x25519" | grep -v "_\?edwards25519_" | grep -v "pqcrystals"
-  else
-    grep -v "^_\?bignum_" | grep -v "_\?curve25519_x25519" | grep -v "_\?edwards25519_" | grep -v "pqcrystals" | grep -v "OPENSSL_armcap_P"
-  fi
+  grep -v "^_\?bignum_" | grep -v "_\?curve25519_x25519" | grep -v "_\?edwards25519_" | grep -v "pqcrystals"
```

Comment:
We shouldn't need to filter pqcrystals anymore I don't think?

https://github.com/aws/aws-lc/blob/main/CMakeLists.txt#L166-L174